andela/hc-leo
# Creates a blog for Healthchecks system

**What does this PR do?**
This PR adds a blog to the healthchecks system

**Description of Task to be completed?**
There is nothing on HealthChecks that allows the tech community to contribute knowledge and expertise on automating business systems. This feature integrates a blog into the system to allow its users to share their acquired knowledge to the tech community. 

**How should this be manually tested?**
- Navigate to ```/blog/``` to check the features added. This is publicly accessible.
- Logged in users can add categories, add, edit and delete blog posts
- To create a new blog post, navigate to  ```/blog/create/```


**What are relevant pivotal tracker stories?**
[#157694808](https://www.pivotaltracker.com/story/show/157694808)

**Screenshots**
> Create a category

<img width="463" alt="screen shot 2018-06-20 at 11 58 58 am" src="https://user-images.githubusercontent.com/25217873/41648494-bc54134c-7481-11e8-995b-9383309dd763.png">

> Create a Blog post

<img width="1440" alt="screen shot 2018-06-20 at 11 58 37 am" src="https://user-images.githubusercontent.com/25217873/41648449-95f7c52c-7481-11e8-8359-a00a858e859c.png">

> Read Blog post

<img width="1440" alt="screen shot 2018-06-20 at 11 59 23 am" src="https://user-images.githubusercontent.com/25217873/41648720-5fb1e8f2-7482-11e8-871a-4071155de180.png">

> Edit Blog post

<img width="1440" alt="screen shot 2018-06-20 at 11 59 57 am" src="https://user-images.githubusercontent.com/25217873/41648793-962052a2-7482-11e8-8f11-de18681c7c33.png">

> Delete Blog post (*Click on the red delete button*)

<img width="1440" alt="screen shot 2018-06-20 at 11 59 40 am" src="https://user-images.githubusercontent.com/25217873/41648860-bdc41154-7482-11e8-888c-141e72fc7b23.png">
